packages/backend/src/app/helper/system/system.ts

```diff
@@ -6,6 +6,14 @@ export const system = {
     return getEnvVar(prop);
   },
 
+  getNumber(prop: SystemProp): number | undefined {
+    const number = getEnvVar(prop);
+    if(number === undefined) {
+      return undefined;
+    }
+    return Number(number);
```

Comment:
Please note that this line could return a lot of unexpected results, for example:
- `Number('ab')` returns `NaN`
- `Number('0x10')` returns `16`, i.e. parsed as hexadecimal
- `Number('')` returns `0`

Here's one way how this could be handled
```ts
getNumber(prop: SystemProp): number | null {
  const stringNumber = getEnvVar(prop);

  if (!stringNumber) {
    return null;
  }

  const parsedNumber = Number.parseInt(stringNumber, 10);

  if (Number.isNaN(parsedNumber)) {
    return null;
  }

  return parsedNumber;
}
```

sorry, I couldn't resist implementing an example!